src/group_theory/complement.lean

```diff
@@ -406,3 +405,73 @@ begin
 end
 
 end subgroup
+
+namespace subgroup
+
+open equiv function mem_left_transversals mul_action mul_action.quotient zmod
+
+universe u
+
+variables {G : Type u} [group G] (H : subgroup G) (g : G)
+
+/-- Partition `G ⧸ H` into orbits of the action of `g : G`. -/
+noncomputable def quotient_equiv_sigma_zmod : G ⧸ H ≃
+  Σ (q : quotient (orbit_rel (zpowers g) (G ⧸ H))), zmod (minimal_period ((•) g) q.out') :=
+(self_equiv_sigma_orbits (zpowers g) (G ⧸ H)).trans
+  (sigma_congr_right (λ q, orbit_zpowers_equiv g q.out'))
+
+lemma quotient_equiv_sigma_zmod_symm_apply
+  (q : quotient (orbit_rel (zpowers g) (G ⧸ H))) (k : zmod (minimal_period ((•) g) q.out')) :
```

Comment:
Note we now have the alias
```suggestion
  (q : orbit_rel.quotient (zpowers g) (G ⧸ H)) (k : zmod (minimal_period ((•) g) q.out')) :
```
which exists primarily just to enable dot notation. I don't know if any of these lemmas could actually benefit from that, so up to you if you want to rename them.